benchmarks/benchmarks/analysis/distances.py

```diff
@@ -11,120 +17,243 @@
 except:
     pass
 
-class DistancesBench(object):
-    """Benchmarks for MDAnalysis.analysis.distances
-    functions.
+
+class BetweenBench(object):
+    """Benchmark for the MDAnalysis.analysis.distances.between
+    function.
     """
+    u = MDAnalysis.Universe(GRO)
```

Comment:
is it necessary to initialize `u = MDAnalysis.Universe(GRO)` both at the class level and inside the `setup()` of each benchmark class?

Reply:
No you're right, sorry these were holdovers from earlier in the writing process and should have been deleted.